File: examples/python-app/src/python_app/formatter.py
from .processor import Report


def formatReport(report: Report) -> str:
	lines = [
		f"Project: {report.projectName}",
		f"Lines: {report.lineCount}",
		"Filtered:"
	]
	lines.extend([f"- {line}" for line in report.filteredLines])
	return "\n".join(lines)

File: examples/ts-app/src/formatter.ts
import type { Report } from "./processor";

export const formatReport = (report: Report): string => {
	const lines = [
		`Project: ${report.projectName}`,
		`Lines: ${report.lineCount}`,
		`API key configured: ${report.apiKeyConfigured ? "yes" : "no"}`,
		"Filtered:",
	];
	lines.push(...report.filteredLines.map((line) => `- ${line}`));
	return lines.join("\n");
};
